.github/workflows/ci.yml

```diff
@@ -352,6 +352,49 @@ jobs:
               body: 'Failed to pass the UI Test. If this PR is for UI change and the error is snapshot mismatch, please add "ui change" label to the PR for updating the snapshot.'
             })
 
+  build_component_examples:
+    name: Build And Test Component+Binding Examples
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Use Node.js v14.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: '14.x'
+      - name: Install rush
+        run: npm install -g @microsoft/rush@5.47.0
+      - name: Install dependencies
+        run: rush install
+      - name: Build
+        run: rush build -t component-examples
+      - name: Visual Regression Tests
+        id: visualregressiontests
+        run: |
+          cd samples/tests
+          rushx test:e2e:examples
+        env:
+          CONNECTION_STRING: ${{ secrets.CONNECTION_STRING }}
+      - name: Upload snapshot diff
+        if: ${{ always() && steps.visualregressiontests.outcome == 'failure' }}
```

Comment:
What's `always() &&` ?

Reply:
That means it will run even previous steps fail